### ISSUE TITLE
ArchLinux image build issue

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal-testsuite-dockerfiles#45]
@akobel By the way, are you using ArchLinux as your Linux distribution? I don't, and I don't know how to maintain those docker files: I use Fedora/CentOS, and I also know Debian/Ubuntu tools.

Would you volunteer to maintain our ArchLinux images? They were created by Philipp when he was there. I have a build issue with the main ArchLinux image: see this [build log on the Docker Hub].

[build log on the Docker Hub]: https://hub.docker.com/r/cgal/testsuite-docker/builds/bfe9c7arlmxsvufdsjvdctk/

The command is:
```
pacman -Sy --noconfirm && pacman-key --refresh-keys && pacman -Su --noconfirm && pacman-db-upgrade 
```
and the error message is:
```
[...]
downloading which-2.21-2-x86_64.pkg.tar.xz...
checking keyring...
downloading required keys...
:: Import PGP key 2048R/, "Christian Hesse (Arch Linux Package Signing) <arch@eworm.de>", created: 2011-08-12? [Y/n] 
error: key "Christian Hesse (Arch Linux Package Signing) <arch@eworm.de>" could not be imported
:: Import PGP key 4096R/, "Antonio Rojas <arojas@us.es>", created: 2014-10-21? [Y/n] 
error: key "Antonio Rojas <arojas@us.es>" could not be imported 
error: required key missing from keyring
error: failed to commit transaction (unexpected error)
Errors occurred, no packages were upgraded. 
```
See the [build log on the Docker Hub] for more details.

**Comment by akobel** [comment]
@lrineau Yes, I'm using Arch whenever and wherever I can, and I'm perfectly familiar with the Arch side. Fixing the problem is trivial with shell access, but I am absolutely clueless about the Docker side. I tried to get a clue of how the image creation works, and/or how I can enter such a container and supervise the CGAL setup, but didn't get it within a reasonable amount of time.

In this case, I assume that the base image for the package building procedure uses outdated keys - all Arch packages are signed by the maintainers, and if the keys are outdated, the installation fails. I'll submit a Dockerfile in a minute that tries to re-initialize the keyring before installing anything, but it will be untested.

If you have a gentle introduction at the Docker procedure at hand, let me know. I'd volunteer to maintain those scripts for the foreseeable future if I get the grips of that part.

**Comment by akobel** [comment]
Repopulation of the keyring from scratch is now in eb8621c; untested, so please let me know whether it will work on Docker Hub.

**Comment by lrineau** [comment]
Fixed by #44. Thanks.
